### PR TITLE
[EN] Lecture10

### PR DESCRIPTION
**Pull request by mvcaro**
There were 4 lines I wasn't so sure about, which I marked as "(?)" on the text, for lines. Please check lines:
- 4334
- 5012
- 7503
- 7545

### CODE REVIEW

File: docs/en/week10/lecture10.sbv [hunk 1]
```diff
@@ -4310,40 +4310,40 @@ tasks and multiple different data sets
...
-supervised methods
+supervised methods.
 
 0:58:27.840,0:58:33.270
-so finally we found sort of the Achilles
+So finally, we found the Achilles
 
 0:58:30.690,0:58:35.610
-heel of this method like the tics of
+heel of this method, like the tics (?) of
```
Comment (Atcold) [inline docs/en/week10/lecture10.sbv:4334]:
```suggestion
heel of this method, like the jigsaw
```

File: docs/en/week10/lecture10.sbv [hunk 2]
```diff
@@ -4940,118 +4940,118 @@ these features when you perform k-means
...
 1:07:42.000,1:07:46.830
 predicting hashtags or predicting GPS
 
 1:07:44.700,1:07:50.070
-locations and then to evaluate this
+locations, and then to evaluate this
 
 1:07:46.830,1:07:54.150
-feature based by learning linear probe
+feature by learning, say, linear prop (?).
```
Comment (Atcold) [inline docs/en/week10/lecture10.sbv:5012]: To me it sounds like "probe". I'm asking Ishan himself.

File: docs/en/week10/lecture10.sbv [hunk 3]
```diff
@@ -7496,253 +7497,253 @@ phenomenon at all so we did not really
 have to do any special tricks with
 
 1:42:45.810,1:42:52.260
-Bachner we were able to use batch no
+batch norm, we were able to use batch norm
 
 1:42:47.730,1:42:54.930
-masses okay and it's not necessary for
+as it is (?) [Ishan] Okay and it's not necessary for
```
Comment (Atcold) [inline docs/en/week10/lecture10.sbv:7503]: Yeah, he says "as is". But "as it is" is even better.

File: docs/en/week10/lecture10.sbv [hunk 4]
```diff
@@ -7496,253 +7497,253 @@ phenomenon at all so we did not really
...
-is generally very hard anyway okay do
+is generally very hard anyway. [Student] Okay, do
 
 1:43:20.970,1:43:27.090
-you think that Paul paper works with the
+you think that PIRL paper works with the
 
 1:43:25.020,1:43:29.520
-Bashan omnious because it uses a memory
+batch norm layer (?) because it uses a memory
```
Comment (Atcold) [inline docs/en/week10/lecture10.sbv:7545]: "layers" instead of "layer". Why the (?) here? You heard it correctly.

**Review by Atcold — CHANGES_REQUESTED** [review]
Waiting to hear from Ishan for that "probe". It could easily be that the "linear probe" is the "linear classifier" / "logistic regression".

File: docs/en/week10/lecture10.sbv (hunk 2 above)
Comment (mvcaro) [reply docs/en/week10/lecture10.sbv:5012]: Great, thank you!

File: docs/en/week10/lecture10.sbv (hunk 4 above)
Comment (mvcaro) [reply docs/en/week10/lecture10.sbv]: I was missing the S :smile:

**Comment by lbourdois** [comment]
Hi @mvcaro 

Before this PR is merged, would it be possible to modify the timers overlay please ?

An example with the first lines : 

0:00:00.979,**0:00:05.730**
The recording is running and so as you

0:00:03.689,0:00:09.510
can see, today we have a guest lecturer,

**0:00:05.730**,0:00:11.519
we have a Ishan Mishra. Ishan Mishra is a

0:00:09.510,0:00:14.309
research scientist at Facebook AI


I think it's important to do it while you can make changes. It hasn't been done for the files from lectures 08 and 12 and it complicates the translation.
Then afterwards it's just psychological but seeing that there are only about 4000 lines instead of 8000 to translate, it's easier to recruit people to translate.

**Review by Atcold — APPROVED** [review]
Thanks!

**Pull request merged**